containers/src/Data/Sequence/Internal.hs

```diff
@@ -199,7 +199,7 @@ import Prelude hiding (
 #if MIN_VERSION_base(4,11,0)
     (<>),
 #endif
-    Applicative, (<$>), foldMap, Monoid,
+    Applicative(..), (<$>), foldMap, Monoid,
```

Comment:
~~Expand `..` to precise list of imports.~~
Maybe there is a solution not needing a `(..)` import?